### ISSUE TITLE
Use head block instead of the deprecated devel

### ISSUE THREAD

**Issue by adamfeldman** [clojure/homebrew-tools#1]
tl;dr: I am making a PR to fix an issue that was caused by a change by Homebrew.

I ran `brew upgrade -n` and got this warning message (it printed out twice):

> Warning: Calling 'devel' blocks in formulae is deprecated! Use 'head' blocks or @-versioned formulae instead.
Please report this issue to the clojure/tools tap (not Homebrew/brew or Homebrew/core), or even better, submit a PR to fix it:
  /usr/local/Homebrew/Library/Taps/clojure/homebrew-tools/Formula/clojure.rb:7

Homebrew seems to be [deprecating `devel` blocks](https://github.com/Homebrew/brew/commit/22857b56b9ff732c83f9bde3b58c3d579ac7f3b1). 

[AFAICT](https://github.com/dart-lang/homebrew-dart/pull/78), `head` blocks are the recommended replacement.

The [Homebrew docs](https://docs.brew.sh/Formula-Cookbook#unstable-versions-devel-head)* describe `head` and `devel` blocks:

> head URLs (activated by passing --HEAD) build the development cutting edge.
> ...
> The devel spec (activated by passing --devel) is used for a project’s unstable releases. devel specs are not allowed in Homebrew/homebrew-core.

`clojure/homebrew-tools` currently uses the following command to install a dev release:

```brew install --devel clojure/tools/clojure```

As a result of Homebrew's changes, this is replaced with:

```brew install --head clojure/tools/clojure```

*A casual read of the docs makes it appear that `head` uses a different syntax than `devel`. A quick review of the [Homebrew rubydocs](https://www.rubydoc.info/github/Homebrew/brew/Formula#head-class_method), and other [formulas](https://github.com/Homebrew/homebrew-core/pull/42727/commits/3c73400f781e05692a0ee77a179472600b29e55f) on Github, implies a simple renaming is all that's needed.

**Comment by include** [comment]
Hi. I am having this same issue but I don't find/know if there is any solution yet.
Any feedback or guide would be nice :) thank you.

**Comment by oldgit** [comment]
Hi all,
The commit 4c2ee07 into the master branch from @puredanger fixes this Homebrew warning issue by removing the`devel` block.
So this issue can now be closed.
